andela/bp-esa-frontend
# 164072314 Redesign dashboard header panel

#### What does this PR do?
Redesigns the dashboard's header

#### Description of Task to be completed?
 - Add responsiveness to the panel header.
 - Modify the panel header to look like the Invision mockups.

#### How should this be manually tested?
1. Start the project and log in to the application.
2. View the dashboard. It should resemble the screenshots below.
3. Test the responsiveness by increasing or reducing the width of the browser window.

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
* [#164072314](https://www.pivotaltracker.com/story/show/164072314)

#### Screenshots (If applicable)
**Before**
![image](https://user-images.githubusercontent.com/41262970/53878747-b831a000-401d-11e9-8cec-962d43365328.png)

**After**
![image](https://user-images.githubusercontent.com/41262970/54122810-23f57d80-440f-11e9-8329-9a6733d84588.png)
![image](https://user-images.githubusercontent.com/41262970/54122852-3ec7f200-440f-11e9-966f-8066338e5292.png)
